crypto/keys/internal/ecdsa/privkey.go

```diff
@@ -9,7 +9,56 @@ import (
 	"math/big"
 )
 
-// GenPrivKey generates a new secp256r1 private key. It uses operating system randomness.
+// p256Order returns the curve order for the secp256r1 curve
+// NOTE: this is specific to the secp256r1/P256 curve,
+// and not taken from the domain params for the key itself
+// (which would be a more generic approach for all EC)
+// In *here* I don't need to do it as a method on the key
+// since this code is only called for secp256r1
+// if called on a key:
+// func (sk PrivKey) pCurveOrder() *.big.Int {
+//     return sk.Curve.Params().N
+// }
```

Comment:
We don't use first person ("I") in code comments. Let's change to "we".
Also, I think adding a commented function is a bit misleading.

Reply:
Fixed in latest commits